src/components/modals/ActionScorer.tsx

```diff
@@ -125,6 +128,30 @@ function getColumns(intl: InjectedIntl, hideScore: boolean): IRenderableColumn[]
             isResizable: true,
             isSorted: true,
             isSortedDescending: true,
+            getSortValue: (scoredBase, component) => {
+                // If the current action is the same as the new give it high value to put it at the top
+                if (component.state.newAction && component.state.newAction.actionId === scoredBase.actionId) {
+                    return 100
+                } 
+
+                // TODO: Fix type so we can use typed property access
+                const score = scoredBase['score']
+
+                // If score base does not have score it's either not scorable or not available
+                // prioritize not scorable over not available but both at bottom of list
+                if (!score) {
+                    if (scoredBase['reason'] === ScoreReason.NotAvailable) {
+                        return -100;
+                    } else {
+                        let isAvailable = component.isUnscoredActionAvailable(scoredBase as UnscoredAction);
```

Comment:
This is the part that is really weird to me.

The server is returning objects that have `reason: notScorable` and also `reason: notAvailable` and yet even for actions that are 'notScorable' we classify them as "available" or "not available" which overlaps with the actual `notAvailable` from service.

Reply:
A "notScorable" action may be available or not - the server is running an older model (i.e. training still taking place) so it doesn't know.  In that case the client does the calculation to determine which category it is in